### ISSUE TITLE
Post Install Message

### ISSUE THREAD

**Issue by thisMagpie** [thisMagpie/fftw#18]
I notice the post install message never actually seems to print to the terminal... What sort of thing would cause that to happen and should this be a concern or am I just not scrolling back far enough?

https://github.com/thisMagpie/fftw/blob/master/nmatrix-fftw.gemspec#L36

**Comment by cjfuller** [comment]
I don't think it should print unless you're doing `gem install fftw`, which given its absence on rubygems.org, I presume you're not doing yet.

**Comment by thisMagpie** [comment]
@cjfuller the nmatrix post install works with the github repo, I think.It seems to be invoked by

```
Gem::Specification 
```

stuff.

**Comment by cjfuller** [comment]
I suppose it would depend on which commands you're running.  I'm usually doing `rake compile`, which doesn't show the message, but other commands that involve the gemspec might.

**Comment by agarie** [comment]
Don't worry about it. `rake compile`, as Colin said, won't print the post
install message anyway.

However, if you bundle the repo as a gem locally and install from that gem
(`rake repackage && gem install pkg/*.gem`, if I remember correctly), it
should be printed.

**Comment by thisMagpie** [comment]
I was just calling it in Rakefile but then I realised calling rake for any task would execute the message so I removed that functionality because it did not make sense for most rake tasks.

I just found it's interesting/concerning that this call doesn't happen with a `gemspec` call in Gemfile alone since there is some set up stuff going on in there too, which is stuff I want to be read but I guess I can be less worried now since fftw.gemspec is no longer managing dependencies and now I don't need to find nmatrix headers and libraries as much as I thought I would before.

I guess the answer must be somewhere in rake judging by what you are both saying @agarie  and @cjfuller, I'll hold this open until I am more certain I can pin down the process which leads to the execution of that message for sure though, I think.
